exporter/observiqexporter/exporter.go

```diff
@@ -15,9 +15,41 @@
 package observiqexporter
 
 import (
+	"errors"
+
 	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/exporter/exporterhelper"
 )
 
 func newObservIQLogExporter(config *Config, params component.ExporterCreateSettings) (component.LogsExporter, error) {
-	return nil, nil
+	if config == nil {
+		return nil, errors.New("config must not be nil")
+	}
+
+	if err := config.validateConfig(); err != nil {
+		return nil, err
+	}
+
+	client, err := buildClient(config, params.Logger, defaultClock)
+	if err != nil {
+		return nil, err
+	}
+
+	exporter, err := exporterhelper.NewLogsExporter(
+		config,
+		params.Logger,
+		client.sendLogs,
+		// explicitly disable since we rely on http.Client timeout logic.
+		exporterhelper.WithTimeout(exporterhelper.TimeoutSettings{Timeout: 0}),
+		exporterhelper.WithRetry(config.RetrySettings),
+		exporterhelper.WithQueue(config.QueueSettings),
+		exporterhelper.WithStart(client.start),
```

Comment:
Assuming you're talking about the WithTimeout -- this is something I'm unsure of, actually. The splunkhecexporter has this line, and the comment above the function in exporterhelper hints that without specifying WithTimeout it will timeout in 5 seconds. So I'm unsure exactly what is expected within the code base for this case.

Reply:
Disregard that last comment, I see the issue with WithStart being just a noop. Should be removed now.

Reply:
No, I meant `WithStart`, or rather that `client.Start` doesn't do anything, so I doubt `WithStart` is doing anything (since it is an option).